client/src/artist/ArtistPage.js

```diff
@@ -1,8 +1,9 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
+import { motion } from 'framer-motion'
 import ArtistPreview from './ArtistPreview'
 import UnmuteMessage from '../app/UnmuteMessage'
-import Navigation from '../app/Navigation'
```

Comment:
Oh, no navigation at all required any more? Super interesting. :)

Reply:
I outsourced the navigation to the app because it's not part of a single page and so it led to animation errors.